include/ruby/random.h

```diff
@@ -84,9 +114,20 @@ typedef struct {
     /** Number of bits of seed numbers. */
     size_t default_seed_bits;
 
-    /** Initialiser function. */
+    /**
+     * Lower 16-bit: major/minor version
+     * Higher bits:  flags
+     *
+     * This is `size_t` because of padding, but not a size of anything.
+     */
+    size_t version_flags;
```

Comment:
It feels much cleaner to use two `uint16_t` fields for this (or even three: uint8_t major, uint8_t minor, uint16_t flags)